### ISSUE TITLE
[QUESTION] Is this repository (or OpenShift Runner Images) still maintained at all?

### ISSUE THREAD

**Issue by rafaelrezend** [redhat-actions/openshift-actions-runners#31]
### Question

I see no activities here since over an year... The [Runner images in quay.io](https://quay.io/repository/redhat-github-actions/runner?tab=tags) were last released in April 2022...
It all sounds very risky to still rely on this, but OpenShift is an Enterprise-grade solution after all, right?

If this isn't maintained, what is the current _Red Hat_ recommendation regarding self-hosted GitHub Runners in OpenShift?

**Comment by BugsBuggy** [comment]
Any updates so far?

**Comment by mkubenka** [comment]
You’re right that the old OpenShift Runner images haven’t been updated since April 2022, so relying on them long-term can be risky. Personally, I recommend moving to the [Actions Runner Controller (ARC)](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller) for self-hosted GitHub Runners on Kubernetes/OpenShift.

**Why ARC?**  

- It’s actively maintained by GitHub and the community.  
- Provides built-in autoscaling, health checks, and easier upgrades.  

**OpenShift-specific setup**  

For an OpenShift deployment, you’ll need to adjust your security and configuration settings. A great reference is the Red Hat Developer article “How to securely deploy GitHub ARC on OpenShift” (Feb 17, 2025):  
- Sets up the right ServiceAccount and RoleBindings  
- Configures PodSecurityPolicies (or SCCs) for runner pods  
- Integrates with OpenShift’s internal image registry  

🔗 https://developers.redhat.com/articles/2025/02/17/how-securely-deploy-github-arc-openshift

**Hands-on walkthrough**  
If you’d like a step-by-step tutorial, check out [Natalie’s](https://github.com/some-natalie) “Kubernoodles, Pt. 5: Self-Hosted GitHub Runners on OpenShift” blog post:  

🔗 https://some-natalie.dev/blog/kubernoodles-pt-5/
